<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferWindow</class>
 <widget class="QMainWindow" name="TransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QLabel" name="contLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cont</string>
    </property>
   </widget>
   <widget class="QLabel" name="disponibilLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>disponibil</string>
    </property>
   </widget>
   <widget class="QLabel" name="inContLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>In contul:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inContEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nume benficiar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mesajLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesaj:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mesajEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="valoareEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="valoareLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Valoare:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Accepta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renuntaButon">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Renunta</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
